packages/overdrive/lib/components/private/CheckableBase/CheckableBase.tsx

```diff
@@ -43,29 +43,63 @@ export const CheckableBase: FunctionComponent<Props> = ({
 	const nakedLabel = ['string', 'number'].includes(typeof label);
 
 	return (
-		<Box className={clsx(styles.root, styles.tappable, className)}>
-			<input
+		<Box
```

Comment:
We need reinstate the _hover_ state for each of the items. And perhaps also remove the excesive padding between items.

![image](https://user-images.githubusercontent.com/599459/88354207-8ac8cc00-cda3-11ea-9255-501c5d5a1292.png)

Reply:
Done